### PR TITLE
Add test_json_map.py and default_map.json

### PR DESCRIPTION
**Pull request by ll7**

### CODE REVIEW

**Comment by Bonifatius94** [comment]
implement an officially supported map file format instead of this example

**Comment by ll7** [comment]
So we should research typical map file formats?

**Comment by Bonifatius94** [comment]
Your format is fine, but I'd like to move the parser function to map\_config.py.

We should also consider to change the map\_definition parameter of Simulator\_v2 to either MapDefinition or string such that the simulator can figure out the parsing internally.

**Comment by ll7** [comment]
We could consider handling maps as svg files, which would reduce the importance of a map editor. Drawing svg maps, obstacles and routes could be easier and different classes are identified by color coding?

https://www.w3.org/wiki/SVG_Map could be helpful, but SVG itself is just a standardized XML file, which you dislike?

**Comment by Bonifatius94** [comment]
SVG Map is a very bloated format. It's very hard to parse and manually edit. The only advantage would be that OpenStreetMap uses this format, so you can get access to real map data. But even then, proportions are not 1:1.

**Comment by Bonifatius94** [comment]
I've added your default map and map parser logic into the project and pushed it to main. We can think about official map formats at a later stage of this project. For now, I'll close your PR. Thank you for your contribution.
